src/libs/actions/Subscription.ts

```diff
@@ -231,4 +231,60 @@ function clearUpdateSubscriptionSizeError() {
     });
 }
 
-export {openSubscriptionPage, updateSubscriptionAutoRenew, updateSubscriptionAddNewUsersAutomatically, updateSubscriptionSize, clearUpdateSubscriptionSizeError, updateSubscriptionType};
+function clearOutstandingBalance() {
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.SUBSCRIPTION_RETRY_BILLING_STATUS_PENDING,
```

Comment:
do we not want to set error: false? and success: false?

Reply:
success has to stay the way it is now but I will look into error. I doubt we have to change it but we'll see